.env.example

```diff
@@ -4,4 +4,7 @@ DATABASE_PASSWORD=postgres
 DATABASE_PORT=5432
 DATABASE_NAME=postgres
 
-STATIC_WORKER_POOL=5
\ No newline at end of file
+STATIC_WORKER_POOL=5
+WHITE_LIST_PATH=/home/user/go/src/github.com/ufcg-lsd/arrebol-pb/whitelist
+ARREBOL_PRV_KEY_PATH=/home/user/arrebol_key
```

Comment:
Let's use priv, for standardization purposes

Reply:
Done.